### DISCUSSION TITLE
Add metadata to pick list items

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5896] [Feature Requests]
'Usage notes' for a pick list have been requested numerous times over the years. Right now, to store metadata with a pick list item, we need to create an entire table to capture it (see `preptype` so that we can indicate if something is loanable for business logic).

We could extend the `picklistitem` table to include a description field and a few boolean fields. These can then be looked at by business logic and allow users to display the pick list description in the UI.

**Comment by maxpatiiuk** [comment]
an important note here is performance consideration.
some institutions have VERY large pick lists (i.e CSIRO and others).
even if empty, fetching extra database column over request could make those slow cases even slower

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8216
